bundles/org.codechecker.eclipse.plugin/src/org/codechecker/eclipse/plugin/views/console/ConsoleFactory.java

```diff
@@ -23,6 +23,18 @@ private static MessageConsole getConsole() {
 
     public static void consoleWrite(String msg) {
         console = getConsole();
+        if (console != null) {
+            IConsoleManager manager = ConsolePlugin.getDefault().getConsoleManager();
+            IConsole[] existing = manager.getConsoles();
+            boolean exists = false;
+            for (int i = 0; i < existing.length; i++) {
+                if (console == existing[i])
+                    exists = true;
+            }
+            if (!exists)
+                manager.addConsoles(new IConsole[] { console });
+            manager.showConsoleView(console);
+        }
```

Comment:
This block is duplicated in lines https://github.com/Ericsson/CodeCheckerEclipsePlugin/blob/090afb9378e858a3c7d5f04f3e6b9cda9aa8c9b8/bundles/org.codechecker.eclipse.plugin/src/org/codechecker/eclipse/plugin/views/console/ConsoleFactory.java#L51-L63. These should be refactored into a common static method, and used in both places.